docs/source/examples/nnclr.rst

```diff
@@ -3,7 +3,33 @@
 NNCLR
 =====
 
-Example implementation of the NNCLR architecture.
+NNCLR is a self-supervised framework for visual representation learning that builds upon contrastive methods. It shares similarities with SimCLR, such as using two augmented views of the same image, projection and prediction heads, and a contrastive loss. However, it introduces key modifications:
+
+1. Nearest Neighbor Replacement: Instead of directly comparing two augmented views of the same sample, NNCLR replaces each sample with its nearest neighbor in a support set (or memory bank). This increases semantic variation in the learned representations.
+2. Symmetric Loss: The contrastive loss is made symmetric to improve training stability.
+3. Architectural Adjustments: NNCLR employs different sizes for projection and prediction head layers compared to SimCLR.
+
+These improvements result in significantly better performance across multiple self-supervised learning benchmarks. Compared to SimCLR and other self-supervised methods, NNCLR achieves:
+- Higher ImageNet linear evaluation accuracy.
+- Improved semi-supervised learning results.
+- Superior performance on transfer learning tasks, outperforming BYOL, SimCLR, and even supervised ImageNet pretraining in 8 out of 12 benchmarked cases.
```

Comment:
Optional: Leave this section away.

Reply:
I would keep it. But the list is not rendered correctly.

Reply:
![image](https://github.com/user-attachments/assets/9eae6669-f762-4053-86b5-5c6ecc6af03b)